app/inbound/save-settlement.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../data')
+const getPaymentRequestByInvoiceNumber = require('./get-payment-request-by-invoice-number')
+
+const saveSettlement = async (settlement, transaction) => {
+  const invoiceNumber = settlement.invoiceNumber
+  const paymentRequest = await getPaymentRequestByInvoiceNumber(invoiceNumber, transaction)
+  settlement.paymentRequestId = paymentRequest.paymentRequestId
+  settlement.schemeId = paymentRequest.schemeId
```

Comment:
If we're removing the FK constraint, we can probably remove this from here and just save the settlement.

Reply:
updated as suggested